src/main/kotlin/com/terraformation/backend/email/WebAppUrls.kt

```diff
@@ -103,7 +103,7 @@ class WebAppUrls(
   }
 
   fun acceleratorConsoleApplication(applicationId: ApplicationId): URI {
```

Comment:
Can you rename this to `fullAcceleratorConsoleApplication`? We try to use the `full` prefix on methods that return full URLs rather than just paths.

Reply:
Sure! I will create a new method for this, because I think that for the notification it's ok to use just the path